src/unix/linux_like/android/mod.rs

```diff
@@ -2349,6 +2349,20 @@ f! {
     pub fn SO_EE_OFFENDER(ee: *const ::sock_extended_err) -> *mut ::sockaddr {
         ee.offset(1) as *mut ::sockaddr
     }
+
+    // Sadly, Android before 5.0 (API level 21), the accept4 syscall is not
+    // exposed by the libc. As work-around, we implement it through `syscall`
+    // directly. This workaround can be removed if the minimum version of
+    // Android is bumped. When the workaround is removed, `accept4` can be
+    // moved back to `linux_like/mod.rs`
+    pub unsafe fn accept4(
```

Comment:
```suggestion
    pub fn accept4(
```

Reply:
Is that the right solution? All the other syscall wrappers are implicitly unsafe because they're extern "C" declarations.

I imagine it would cause warnings about unnecessary `unsafe` blocks for users of `accept4()` when compiling for Android platforms.

Reply:
Oh, the `f` macro already takes care of that, it seems. Applying the suggestion :)